contracts/vault/ERC4626Strict.sol

```diff
@@ -24,7 +24,7 @@ contract ERC4626Strict is ERC4626Upgradeable, TetuERC165 {
   // *************************************************************
 
   /// @dev Connected strategy. Can not be changed.
-  IStrategyStrict public immutable strategy;
```

Comment:
keep strategy immutable and set vault via init for the strategy

Reply:
done